datajunction-server/datajunction_server/database/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""All database schemas."""
```

Comment:
nit: are these database schemas as in namespaces or as in table structures? I think the latter but the phrase `database schemas` is confusing. I think `table schemas` would be clearer.